(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'int' object has no attribute 'get'"
p3
sS'layer'
p4
S'/home/bemobi/Downloads/web2py/applications/touch_pay_client/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f9934d7a9b0, file "/home/bemobi/Downloads/web2py/applications/touch_pay_client/controllers/default.py", line 10>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/bemobi/Downloads/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/bemobi/Downloads/web2py/applications/touch_pay_client/controllers/default.py", line 71, in <module>\n  File "/home/bemobi/Downloads/web2py/gluon/globals.py", line 409, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/bemobi/Downloads/web2py/applications/touch_pay_client/controllers/default.py", line 28, in find_finger\n    r = requests.get("http://139.82.240.23:8081/walletapi/costumer/%s/" % id)\nAttributeError: \'int\' object has no attribute \'get\'\n'
p11
s.